mapviz/src/mapviz.ui

```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>732</width>
+    <width>600</width>
```

Comment:
Added in inadvertently due to using Qt Designer to edit the .ui file.  It shouldn't actually affect anything.